src/npc_class.h

```diff
@@ -64,6 +64,12 @@ class npc_class
         npc_class_id id;
         bool was_loaded = false;
 
+        Group_tag worn_override;
```

Comment:
In a separate PR you might consider implementing a default item group so we could eliminate more hardcoded item groups from core